General/qodana-yaml.md

```diff
@@ -61,15 +61,15 @@ failThreshold: 0
 profile:
   name: qodana.recommended
 exclude:
-    - name: Annotator
-    - name: AnotherInspectionId
-paths:
-  - relative/path
-  - another/relative/path
-- name: All
-paths:
-  - asm-test/src/main/java/org
-  - asm
-  - benchmarks
-  - tools
-```
\ No newline at end of file
+  - name: Annotator
+  - name: AnotherInspectionId
+    paths:
+      - relative/path
+      - another/relative/path
+      - name: All
+    paths:
+      - asm-test/src/main/java/org
+      - asm
+      - benchmarks
+      - tools
+```
```

Comment:
Thanks for spotting this out!

There is still one indentation issue. 
```
  - name: Annotator
  - name: AnotherInspectionId
    paths:
      - relative/path
      - another/relative/path
   - name: All 
     paths:
      - asm-test/src/main/java/org
      - benchmarks
      - tools
```

Also, let's remove the `asm` path from the example.

Reply:
As your wish: https://github.com/JetBrains/Qodana/pull/13/commits/9fa525cdbe2fc2cba873b8dadd19709e0e2cc875